docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,279 @@
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/num_cvs.png
+    :align: center
+    :scale: 50
+    :alt: Num CVs
+
+    Setting the Number of Control Values
+
+The final values to set are ``TELEM_IDS_1`` and ``TELEM_IDS_2``. These control which module IDs have telemetry requested. When you click on one of these parameters, a list of checkboxes should show up. Tick each module ID that is on your bus to request telemetry from that module.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/telem_ids.png
+    :align: center
+    :scale: 50
+    :alt: Telem Modules
+
+    Setting the Modules to Request Telemetry From
+
+Actuator Setup
+==============
+
+The next step is to set up the actuator associations. Go to the actuator tab in QGroundControl settings and select the 'Vertiq IO' tab on the top right. Initially every ESC function will be disabled. Each ESC number in this section corresponds to a CVI number plus 1. For example ESC 1 corresponds to CVI 0 and so on. Based on the geometry drawing, match the motor with the appropriate ESC # (and therefore CVI). As you do, also ensure that the direciton checkbox is ticked properly. Module ID is irrelevant here.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/geometry_setup.png
+    :align: center
+    :scale: 50
+    :alt: geometry setup
+
+    Matching Motor Geometry to ESCs
+
+In the example image above, the from right module has a ``THROTTLE_CVI`` of 0, the back left module; 1, front left; 2, back left; 3. This is conveniently in order, so an additional example is shown below where this is not the case.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/silly_geometry_setup.png
+    :align: center
+    :scale: 50
+    :alt: geometry setup
+
+    Matching Motor Geometry to ESCs with a 'Non-Conventional' Setup
+
+In the setup above the ``THROTTLE_CVI`` for the front right; 0, back right; 1, front left; 2, back left; 3. As long as the module CVI to Motor number association is correct, the right motor will spin.
+
+Spin Testing
+============
+
+Now that everything is setup correctly, you can attempt to spin the modules. In the same 'Actuators' tab, enable the Acutator Testing slider. Once this is enabled the motors can spin. Slide each motor slide up and down, and ensure that it corresponds to the expected motor in your geometry image.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuator_test.png
+    :align: center
+    :scale: 50
+    :alt: Actuator Test
+
+    Testing the Actuators
```

Comment:
I would love a full setup example of you putting together a quad, and spinning the 4 modules with a realistic setup. Would bring everything together.